docs/dsl.rst

```diff
@@ -1,4 +1,4 @@
-include .special.rst
+.. include:: .special.rst
```

Comment:
This is literally at the top of the page. Yikes
https://nuclear.readthedocs.io/en/latest/dsl.html